olp-cpp-sdk-core/src/client/PendingRequests.cpp

```diff
@@ -28,23 +28,13 @@ namespace {
 constexpr auto kLogTag = "PendingRequests";
 }
 
-PendingRequests::PendingRequests(){};
-
-PendingRequests::~PendingRequests(){};
-
 bool PendingRequests::CancelAll() {
-  RequestMap requests_map;
   ContextMap contexts;
   {
-    std::lock_guard<std::mutex> lk(requests_lock_);
-    requests_map = requests_map_;
+    std::lock_guard<std::mutex> lk(task_contexts_lock_);
```

Comment:
```suggestion
    std::lock_guard<std::mutex> lock(task_contexts_lock_);
```
Same below. Everywere else we use `lock` instead of `lk`.